.github/workflows/release.yml

```
@@ -30,10 +30,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - env:
+        HEAD_REF: ${{ github.head_ref }}
       - run: >
```

Steps must have either `uses` or `run` and this change is syntactically incorrect; you'll have to merge this "step" with the next step to make it work, as well as indent `HEAD_REF` properly:

```suggestion
      - env:
          HEAD_REF: ${{ github.head_ref }}
        run: >
```

---

https://github.com/iterative/cml/pull/1448/commits/35cfca563ee885ff520226cf50f69891a41fc255